tests/integration-tests/helix-payload/osx-helix-payload.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+version='1.0.0-ci'
+
+# Clean the NuGet cache from the previous 1.0.0-ci version of the tool
+# TODO: This might have a better solution: https://github.com/dotnet/xharness/issues/123
+echo "Cleaning the NuGet cache from the previous version of the tool..."
+cache_dirs=`dotnet nuget locals All -l | cut -d':' -f 2 | tr -d ' '`
+while IFS= read -r path; do
```

Comment:
Remind me; did you try `dotnet nuget locals all --clear` and it didn't work?  Seems much safer to let nuget do the deleting.

Reply:
I didn't want to do that because we will share OSX machines with other workloads and we would be purging the cache for everyone so that might be undesirable